src/components/v2/Layout/Sidebar/index.tsx

```diff
@@ -112,7 +115,7 @@ export const SidebarUi: React.FC<ISidebarProps> = ({ account }) => {
                       component="span"
                       css={[styles.listItemText, styles.mobileListItemText]}
                     >
-                      {text}
+                      {t(`layout.menuItems.${id}`)}
```

Comment:
Same thing as previous comment about needing to include the text itself inside each menu item.